### PR TITLE
Bump Scala version to 2.13.0-M5

### PR DESCRIPTION
**Pull request by julienrf**

### CODE REVIEW

**Comment by SethTisue** [comment]
```
[error] /home/travis/build/scala/scala-collection-compat/scalafix/output213/src/main/scala/fix/RoughlySrc213.scala:9: value Empty is not a member of object scala.collection.immutable.LazyList
[error]   1 #:: 2 #:: 3 #:: LazyList.Empty
[error]                              ^
[error] /home/travis/build/scala/scala-collection-compat/scalafix/output213/src/main/scala/fix/RoughlySrc213.scala:11: value Empty is not a member of object scala.collection.immutable.LazyList
[error]     case LazyList.Empty => true
[error]                   ^
[error] two errors found
```

**Comment by julienrf** [comment]
Could you review this PR, @MasseGuillaume?

**Comment by MasseGuillaume** [comment]
@SethTisue @julienrf  can you add me as a Collaborator?

**Comment by SethTisue** [comment]
@MasseGuillaume done, once you accept the invitation to https://github.com/orgs/scala/teams/team-scala

File: scalafix/rules/src/main/scala/fix/Roughly.scala [hunk 1]
```diff
@@ -48,6 +48,10 @@ final case class Roughly(index: SemanticdbIndex, config: RoughlyConfig)
     Symbol("_root_.scala.collection.immutable.Stream.append.")
   )
 
+  val streamEmpty = SymbolMatcher.exact(
+    Symbol("_root_.scala.collection.immutable.Stream.Empty.")
```
Comment (MasseGuillaume) [inline scalafix/rules/src/main/scala/fix/Roughly.scala]: Can you add it to the scalafix 0.6 branch?

```scala
val streamEmpty = exact(
  "scala/collection/immutable/Stream#empty()."
)
```

https://github.com/scala/scala-collection-compat/blob/scalafix-0.6/scalafix/rules/src/main/scala/fix/Roughly.scala#L44-L46

**Pull request merged**